packages/modules/src/scoring/Scoring.ts

```diff
@@ -0,0 +1,68 @@
+import { SCORE_MODULUS, IJournalContent,IRatingInput,CordKeyringPair } from '@cord.network/types'
+import { Crypto } from '@cord.network/utils'
+import {
+  SCORE_IDENT,
+  SCORE_PREFIX,
+  ScoreType,
+  IScoreAverageDetails,
+} from '@cord.network/types'
+import { Identifier } from '@cord.network/utils'
+import { fetchScore } from './Scoring.chain'
+
+export function adjustAndRoundRating(rating: number): number {
+  rating = Math.round(rating * SCORE_MODULUS)
+  return rating
+}
+
+export function ComputeActualRating(rating: number): number {
+  return rating / SCORE_MODULUS
+}
+
+export function ComputeAverageRating(rating: number, count: number): number {
+  return rating / count
+}
+
+export function generateDigestFromJournalContent(
```

Comment:
generateDigestFromJournalContent and getUriForScore should combine to make a function like fromJournalContent that returns a digest and identifier. These values can be stored by the proposer locally and can be used to compare the on-chain values. If all goes well, the chain will also produce the same identifier.

Reply:
Also we need to process the entity input to create an identifier for the entity. This should also be returned as it's required by the toChain function